pkg/k8s/apis/cilium.io/client/crds/v2alpha1/ciliumworldcidrsets.yaml

```diff
@@ -0,0 +1,86 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.11.4
+  creationTimestamp: null
+  name: ciliumworldcidrsets.cilium.io
+spec:
+  group: cilium.io
+  names:
+    categories:
+    - cilium
+    kind: CiliumWorldCIDRSet
+    listKind: CiliumWorldCIDRSetList
+    plural: ciliumworldcidrsets
+    singular: ciliumworldcidrset
+  scope: Cluster
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .spec.encapsulate
+      name: Encapsulate
+      type: boolean
+    - jsonPath: .spec.cidrs
+      name: CIDRs
+      type: string
+    - jsonPath: .metadata.creationTimestamp
+      name: Age
+      type: date
+    name: v2alpha1
+    schema:
+      openAPIV3Schema:
+        description: CiliumWorldCIDRSet is a set of CIDRs and their routing destination.
+          It is used in certain modes (e.g. highscale-IPCache) to indicate whether
+          destinations require encapsulation.
```

Comment:
nit: the use of "encapsulation" can be a bit ambiguous, especially in the context of networking software. Might be clearer to specify _what_ encapsulation we're talking about here (i.e. vxlan overlay?)